### ISSUE TITLE
v2: crash if a number is used as a key

### ISSUE THREAD

**Issue by warpdesign** [onury/docma#69]
While porting a v1 template to docma v2 I stumbled upon this [crash](https://github.com/onury/docma/issues/55).

After some digging it appears that what makes it crash is the following:

```js
const obj = {
  43: "bar"
};
```

Replacing it with the following code works as a workaround:

```js
const obj = {
  "43": "bar"
};
```

That may be fixed by v3 already but I cannot verify it since I'm focused on making my template work on Docma v2 first.

**Comment by warpdesign** [comment]
The changelog page also mentions that the getSymbolByName signatured has changed:
> BREAKING: Docma utility methods are moved to DocmaWeb.Utils static namespace (formerly under docma.utils).
> DocmaWeb.Utils.getSymbolByName() signature is changed.

Would be great to tell what has changed :)

**Comment by onury** [comment]
You could either check out the [code](https://github.com/onury/docma/blob/master/lib/web/DocmaWeb.Utils.js#L201) or the [documentation](https://onury.io/docma/api/web/utils#DocmaWeb.Utils.getSymbolByName) and/or use some sort of code-completion / intelli-sense. Code is documented so it should give you much proper details about the signature and more; rather than explaining every little thing in the change-log.

**Comment by onury** [comment]
Error you've mentioned is fixed with [v3.0.0](https://onury.io/docma/changelog#300--2018-11-18-). Re-open if you still encounter this with v3+
